modules/tensor_mechanics/include/materials/ComputeIsotropicElasticityTensor.h

```diff
@@ -38,6 +37,9 @@ class ComputeIsotropicElasticityTensor : public ComputeElasticityTensorBase
 
   /// Individual elasticity tensor
   RankFourTensor _Cijkl;
+
+  /// Flag for variable elasticity isotropic tensors
+  MaterialProperty<bool> & _elasticity_tensor_is_constant;
```

Comment:
i agree this would be better not to be a MaterialProperty - just a single bool would be better.